### DISCUSSION TITLE
Exporting items never ends

### DISCUSSION THREAD

**Discussion by mndldm** [retorquere/zotero-better-bibtex#3010] [Q&A] [answered]
After pressing right click on my library, selecting export collection and format: better Bibtex the export just doesn't stop. I've exported my library like this a few time before and I never had any issues. 
![Screenshot 2024-10-02 092425](https://github.com/user-attachments/assets/02394616-4af6-42a0-a55a-f111fc6c8af5)
![Screenshot 2024-10-02 093222](https://github.com/user-attachments/assets/bd530af2-8263-4da1-9ae0-914a5663c288)

**Comment by retorquere** [comment, marked as answer]
Of you want something fixed, open an issue, not a discussion.
